imports/plugins/core/versions/server/migrations/33_change_products_template_name.js

```diff
@@ -0,0 +1,40 @@
+import { Migrations } from "meteor/percolate:migrations";
+import { Packages } from "/lib/collections";
+
+Migrations.add({
+  version: 33,
+  up() {
+    Packages.update({
+      "layout.structure.template": "products"
+    }, {
+      $set: {
+        "layout.$.structure.template": "Products"
+      }
+    }, { bypassCollection2: true });
+
+    Packages.update({
+      "registry.template": "products"
+    }, {
+      $set: {
+        "registry.$.template": "Products"
+      }
+    }, { bypassCollection2: true });
+  },
+  down() {
+    Packages.update({
+      "layout.structure.template": "Products"
+    }, {
+      $set: {
+        "layout.$.structure.template": "products"
+      }
+    }, { bypassCollection2: true });
+
+    Packages.update({
+      "registry.template": "Products"
+    }, {
+      $set: {
+        "registry.$.template": "products"
+      }
+    }, { bypassCollection2: true });
+  }
+});
```

Comment:
These all need to be multi. Change `{ bypassCollection2: true }` to `{ bypassCollection2: true, multi: true }`